compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -1076,6 +1076,21 @@ pub struct VarDebugInfo<'tcx> {
 // BasicBlock
 
 rustc_index::newtype_index! {
+    /// The unit of the MIR [control-flow graph][CFG].
```

Comment:
"unit" is an interesting choice here. Usually in a graph we speak of nodes and edges, so would "node" make more sense here?

Reply:
Good point! I said "unit" because that's what the [rustc-dev-guide says](https://rustc-dev-guide.rust-lang.org/mir/index.html#key-mir-vocabulary):

> Basic blocks: units of the control-flow graph, consisting of: